code/script.js

```diff
@@ -1,47 +1,183 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const chat = document.getElementById('chat');
+const inputWrapper = document.getElementById('input-wrapper');
+const nameForm = document.getElementById('name-form');
+const input = document.getElementById('input');
+const sendBtn = document.getElementById('send-btn');
 
-// Global variables, if you need any, declared here
+// GLOBAL VARIABLES
+// Declaring the question number that will change
+let questionNumber = '';
```

Comment:
You will store numbers here, so you shouldn't initialize it as an empty string. A better option, in my opinion, would be `0`